app/controllers/workflows_controller.rb

```diff
@@ -21,7 +21,6 @@ class WorkflowsController < ApplicationController
 
   # @todo These actions should be extracted from the controller, and instead be handled by an object invoked
   #       by the task
-  include Tasks::AddSpikedInControlHandler
```

Comment:
A lot of the pipelines work by calling methods back on the controller, resulting in a very fat controller. As part of this work I've been taking steps away from that, although have mainly concerned myself with the tasks I've been actively working on, and those that are no-longer used.

- Spiked in control handler no-longer relies on any methods injected in the controller
- Didn't quite get the changes to fully decouple SetDescriptorsHandler unfortunately
- SetCharacterisationDescriptorsHandler is related to the removed task